dev/com.ibm.ws.kernel.metatype.helper/resources/OSGI-INF/l10n/fileset.properties

```diff
@@ -31,4 +31,4 @@ fileset.attr.excludes.name=Excludes pattern
 fileset.attr.excludes.desc=The comma or space separated list of file name patterns to exclude from the search results, by default no files are excluded.
 #monitor
 fileset.attr.scan.name=Scan interval
-fileset.attr.scan.desc=Scanning interval to check the fileset for changes as a long with a time unit suffix h-hour, m-minute, s-second, ms-millisecond (e.g. 2ms or 5s). Disabled (scanInterval=0) by default.
+fileset.attr.scan.desc=The scanning interval to determine whether files are added or removed from the fileset. The individual files are not scanned. The suffix for the interval of time is h-hour, m-minute, s-second, and ms-millisecond, for example, 2ms or 5s. The scanning interval is disabled by default and is disabled manually by setting the scan interval, scanInterval, to 0.
```

Comment:
As per the latest discussion with Karen and Daniel. We suggest the text:  
“fileset.attr.scan.desc=The scanning interval to determine whether files are added or removed from the fileset. The individual files are not scanned. The scanning interval is disabled by default and is disabled manually by setting the scan interval, scanInterval, to 0.”